bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/forms2/util/FormUtil.java

```diff
@@ -0,0 +1,20 @@
+package org.bahmni.module.bahmnicore.forms2.util;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
+public class FormUtil {
```

Comment:
for both the methods, using a regex would have been a much better solution.

Reply:
sure, will check.